### ISSUE TITLE
Use of modern CSS features (Grid, Flex, ...)

### ISSUE THREAD

**Issue by 5ebastianMeier** [process-engine/bpmn-studio#164]
Currently we're avoiding modern CSS features because of a backward compatibility for IE11 (there might be other reasons, but that's the only one I am aware of).

Microsoft has officially dropped support for IE11 a long time ago (https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support). Because of that I would like to drop this support requirement for the web application.

I realize there are enterprise scenarios where IE11 is enforced as the main browser in a whole landscape. In those cases Electron comes in handy as it brings its own version of Chromium, independent from any system boundaries the enterprise landscape might enforce.

**This would really mean a great deal to us.**

Features such as Grid and Flex allow for dramatically easier and more robust code for layouting scenarios.

Usage statistics on IE11 are already down to a minimum with Microsoft strongly pushing Windows 10 to displace older Windows versions and by that effectively the browsers as well.

_See https://caniuse.com/usage-table for reference_

Taking all of this into account, I don't think IE11 can be viewed as a first class use case of the BPMN-Studio. In my opinion the electron fallback is sufficient to compensate for full IE11 support.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by cmg-dev** [comment]
@5ebastianMeier I agree on this.

Please note, that caniuse.com is in my experience no reliable source!

We still need careful front end tests, but that is a different story.

**TL;DR**

**If @moellenbeck - as product owner - is okay with this, we should update the product requirements.**

**Comment by cmg-dev** [comment]
But also note:

**THIS IS NOT ABOUT TO AGREEINGTO THE FANCY NEW CSS FEATURES!**

Everything related to this, has to be aligned with @rrrene.

I will not buy any CSS fanciness just for the sake of being fancy!

**Comment by 5ebastianMeier** [comment]
@cmg-dev I agree with that. This issue was meant as a discussion point about whether or not IE11 should be treated as a first class citizen.

I'm mainly arguing that we will likely drop the IE11 support in the not so far future anyway (because of the outdated nature explained above), which is why we shouldn't build everything with IE11 in mind.

**Comment by 5ebastianMeier** [comment]
More reliable sources:
http://gs.statcounter.com/browser-version-market-share
https://www.w3counter.com/globalstats.php
